e2e/Sandbox/MainWindow.xaml.cs

```diff
@@ -51,6 +52,7 @@ public partial class MainWindow : Window
             new RestDataSourceDashboard(),
             new SalesDashboard(),
             new SqlServerDataSourceDashboards(),
+            new PostgresqlDashboard()

```

Comment:
place this in alphabetical order. As this grows it will make it easier to find in the list

Reply:
Updated